tests/unit_tests/test_sda_vector_math.py

```diff
@@ -1,7 +1,7 @@
 import math
 import numpy
 import unittest
-from SDA import VectorMath
+from SDAWithVectorField import VectorMath
```

Comment:
You should create a separate vector math test case for your package, and leave the other unit tests in operation.